Spigot-API-Patches/0201-Add-no-tick-view-distance-API.patch

```diff
@@ -0,0 +1,88 @@
+From d7cee11536568899a076e0d79db8835e514d9fb8 Mon Sep 17 00:00:00 2001
+From: Mariell Hoversholm <proximyst@proximyst.com>
```

Comment:
Shouldn't this name spotted as the proper author then?

Reply:
Yes. If you desire recognition for porting it you should put that in the patch notes in the header.

Reply:
He's been set as the author